client/source/App.jsx

```diff
@@ -28,16 +29,19 @@ const App = () => {
       });
   }
 
+  const [prodID, setProdID] = useState('product id');
 
   return (
     <div>
-      <Header />
-      <Overview />
-      <button onClick={testGetRequest}>TEST GET PRODUCTS REQUEST</button>
-      <RelatedList />
-      <OutfitList />
-      <QAModule />
-      <ReviewModule />
+      <ProdContext.Provider value={{prodID, setProdID}}>
```

Comment:
Have you thought about moving the `.Provider` method to the index.jsx file so it only needs to wrap around the <App /> component and there will be less merge conflicts as we rearrange the `app.jsx` file